src/runtime_src/core/edge/user/device_linux.h

```diff
@@ -35,6 +35,9 @@ class device_linux : public shim<device_edge>
   void
   set_cu_read_range(cuidx_type ip_index, uint32_t start, uint32_t size) override;
 
+  xclGraphHandle
+  open_graph(const xrt::hw_context& hwctx, const char* name, xrt::graph::access_mode am) override;
+
```

Comment:
Remove not needed

```suggestion
```

Reply:
Same here. This function is needed for Shim to manage hardware contexts.